model/src/main/java/com/epam/esm/model/CustomPage.java

```diff
@@ -0,0 +1,21 @@
+package com.epam.esm.model;
+
+import lombok.Data;
+
+@Data
+public class CustomPage {
+    private static final int MIN_PAGE_NUMBER = 0;
+    private static final int MAX_PAGE_SIZE = 5;
+    private static final int MIN_PAGE_SIZE = 1;
+    private int pageNumber;
+    private int pageSize;
+    private int pageTotal;
+
+    public CustomPage(int pageNumber,int  pageSize,Long pageTotal) {
```

Comment:
This logic is difficult to perceive,
also not auto-formatted

Reply:
done